content/docs/replication/release/_index.md

```diff
@@ -23,4 +23,8 @@ Description: >
 
 ### Known Issues
 
-There are no known issues in this release.
+| Github ID | Description  |   
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [514](https://github.com/dell/csm/issues/514) | **PowerScale:** When creating a replicated PV in PowerScale, the replicated PV's AzServiceIP property has the target PowerScale endpoint instead of the one defined in the target Storage class.                            |   
+| [515](https://github.com/dell/csm/issues/515) | **PowerScale:** If you failover with an application still running and having a mounted the volume, on the target site we cannot mount the PVC due to : "mount.nfs: Stale file handle".                                      |   
```

Comment:
Line 29: Rewrite: If you failover with an application still running and the volume mounted on the target site, then we cannot mount the PVC due to : "mount.nfs: Stale file handle".

Reply:
fixed